osdconfig/proto/config.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+// this package defines the configuration parameters and interfaces
+// requires to run portwork services
+package proto;
+
+service Spec {
+    rpc GetGlobalSpec(Empty) returns (GlobalConfig) {}
+    rpc SetGlobalSpec(GlobalConfig) returns (Ack) {}
+    rpc GetClusterSpec(Empty) returns (ClusterConfig) {};
+    rpc SetClusterSpec(ClusterConfig) returns (Ack) {};
+    rpc GetNodeSpec(NodeID) returns (NodeConfig) {};
+    rpc SetNodeSpec(NodeConfig) returns (Ack) {};
```

Comment:
nit: not sure if need the semicolon at the end

Reply:
both are okay, but let's remove them. done.